openmc/filter.py

```diff
@@ -531,12 +531,13 @@ def get_pandas_dataframe(self, data_size, summary=None):
         ----------
         data_size : Integral
             The total number of bins in the tally corresponding to this filter
-        summary : None or openmc.Summary
-            An optional Summary object to be used to construct columns for
-            distribcell tally filters (default is None). The geometric
+        distribcell_paths : bool
```

Comment:
I was thinking the same thing. Let's make the distribcell paths the default. Only reason I was leaving it as an option is due to the fact that building the multi-index for the paths isn't particularly efficient, and can be a bit slow for a large distribcell tally (*i.e.*, pick your favorite fuel pin in the full core BEAVRS model). However, the performance hit isn't noticeable for the smaller geometries that probably comprise the majority of use cases at this point. And I'm sure things can be sped up a great deal if this ever becomes a real pinch point in the future.